src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -895,6 +895,18 @@ public int Timeout
                 _timeOutSpecified = true;
             }
         }
+        
+        /// <summary>
+        /// Gets or sets a value indicating whether to return after any one process exits.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Any { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether to return the Process objects after waiting.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter PassThru { get; set; }
```

Comment:
@MartinGC94 Makes sense to add return type in the cmdlet definition?

Reply:
Yeah I think so. That's what we do for other commands with a `PassThru` parameter, like: `Stop-Process`.